e2etests/package.json

```diff
@@ -49,8 +49,12 @@
     "job_client": "mocha --reporter spec test/job_client.js",
     "authentication": "mocha --reporter spec test/authentication.js",
     "provisioning": "cd ../provisioning/e2e && npm run ci && cd ../../e2etests",
-    "modules": "mocha --reporter spec test/modules.js",
-    "phase1_fast": "npm-run-all -p -l twin_disconnect service registry device_acknowledge_tests upload_disconnect authentication job_client",
+    "configurations": "mocha --reporter spec test/configurations.js",
+    "module_crud": "mocha --reporter spec test/module_crud.js",
+    "module_messaging": "mocha --reporter spec test/module_messaging.js",
+    "module_twin": "mocha --reporter spec test/module_twin.js",
+    "module_methods": "mocha --reporter spec test/module_methods.js",
+    "phase1_fast": "npm-run-all -p -l twin_disconnect service registry device_acknowledge_tests upload_disconnect authentication job_client module_crud module_messaging module_twin module_methods",
```

Comment:
>module_crud module_messaging module_twin module_methods [](start = 140, length = 55)

all these running in parallel make me wonder how close we are to throttling limits. will have to monitor timeouts.

Reply:
we may also be close to parallelization limits in npm-run-all.  I'm factoring the modules tests out into a different phase.

---
In reply to: [180592320](https://github.com/Azure/azure-iot-sdk-node/pull/271#discussion_r180592320) [](ancestors = 180592320)